debs/SPECS/4.3.0/wazuh-manager/debian/preinst

```diff
@@ -21,6 +21,9 @@ case "$1" in
     install|upgrade)
 
         if [ "$1" = "upgrade" ]; then
+            if pgrep -f ossec-authd > /dev/null 2>&1; then
```

Comment:
I.e: upgrade from 4.0.0 to 4.3.0, this code kills unnecessarily the `ossec-authd` daemon. This kill must be done only in those cases where none of the `wazuh` or `ossec` control stops works.

Reply:
Done: https://github.com/wazuh/wazuh-packages/pull/1123/commits/c8a4ee15953c804a14a9ef4b3ed827f0092ef037